src/pages/DistributionsPage/DistributionsPage.tsx

```diff
@@ -2,7 +2,7 @@ import assert from 'assert';
 import React, { useState } from 'react';
 
 import { isUserAdmin } from 'lib/users';
-import uniqBy from 'lodash/uniqBy';
+import { uniqBy } from 'lodash';
```

Comment:
i'm not sure if this has improved since i started doing it years ago, but i started importing individual lodash functions because it helped webpack not include the entire lodash lib in the frontend bundle

Reply:
This is still recommended for bundlers in their docs